draft-packages/modal/KaizenDraft/Modal/Presets/InputEditModal.tsx

```diff
@@ -28,6 +28,10 @@ export interface InputEditModalProps {
 
 type InputEditModal = React.FunctionComponent<InputEditModalProps>
 
+/**
+ * {@link https://cultureamp.design/components/modal/#input-edit-modal Guidance} |
```

Comment:
@mcwinter07 Said on slack:
> Actually lets go with leaving that one as is and I can put in a ticket to fix that on the docs page side in a seperate issue, since its more a bug than anything